packages/app/app/stacks/page.tsx

```diff
@@ -0,0 +1,11 @@
+"use client";
+
+import { Empty } from "@/app/stacks/empty";
+
+export default function Page() {
+  return (
+    <div className="w-full max-w-screen-xl px-4 mx-auto">
+      <Empty />
```

Comment:
`Empty` is too generic, maybe we can call it EmptyState or EmptyPage

Reply:
I'll use `EmptyState`